src/custom-element.js

```diff
@@ -1364,6 +1364,9 @@ function createBaseCustomElementClass(win) {
       } else {
         const placeholders = dom.childElementsByAttr(this, 'placeholder');
         for (let i = 0; i < placeholders.length; i++) {
+          if (placeholders[i].getAttribute('placeholder') !== '') {
```

Comment:
? is there a `placeholder=<NotEmpty>` we are supporting that this breaks?

Reply:
@aghassemi Yeah this check is too general. What do you think about checking that the element with the `placeholder` attribute isn't an input field, textarea etc? The existing behavior effectively forbids input elements with placeholders.

Reply:
as discussed offline, let's just check tagname = input